ios/RCTTWVideoModule.m

```diff
@@ -216,8 +226,21 @@ - (void)startCameraCapture:(NSString *)cameraType {
     self.localAudioTrack = [TVILocalAudioTrack trackWithOptions:nil enabled:YES name:@"microphone"];
 }
 
-RCT_EXPORT_METHOD(stopLocalVideo) {
-    [self clearCameraInstance];
+RCT_EXPORT_METHOD(stopLocalVideo:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```

Comment:
🎉 nice - making it into a promise was probably the best improvement we could make here.